errors.go

```diff
@@ -6,6 +6,7 @@ import (
 )
 
 var (
+	ErrInsertRace    = errors.New("insert failed due to data race", j.C("ERR_3cacda3bb50f6d47")) // Concurrent sets can cause race errors, can just try again.
```

Comment:
Jus like `Set`, I would use a single error for inserts and updates. Suggest just using `ErrUpdateRace` instead